src/backend/messages/settings.py

```diff
@@ -781,6 +781,9 @@ class Base(Configuration):
         "sdk_url": "/sdk",
         "api_url": "/api/v1.0",
     }
+    DRIVE_APP_NAME = values.Value(
+        default="Drive", environ_name="DRIVE_APP_NAME", environ_prefix=None
+    )
```

Comment:
What do you think to put the `app_name` within the `DRIVE_CONFIG` dict ?
```suggestion
DRIVE_CONFIG = {
"base_url": values.Value(
default=None, environ_name="DRIVE_BASE_URL", environ_prefix=None
),
"file_url": "/explorer/items/files",
"sdk_url": "/sdk",
"api_url": "/api/v1.0",
"app_name": values.Value(
        default="Drive", environ_name="DRIVE_APP_NAME", environ_prefix=None
    )
}
```